utils/domainstate.mli.c

```diff
@@ -0,0 +1,6 @@
+type t =
```

Comment:
This file should probably have a header too

Reply:
Done.